tests/extensions/test_idu.py

```diff
@@ -0,0 +1,71 @@
+import json
+import unittest
+from os import makedirs
+from typing import List
+import pytest
+import requests
+from parameterized import parameterized
+
+from pystac_monty.extension import MontyExtension
+from pystac_monty.sources.idu import (
+    IDUDataSource,
+    IDUTransformer,
+)
+from tests.conftest import get_data_file
+from tests.extensions.test_monty import CustomValidator
+
+CURRENT_SCHEMA_URI = "https://ifrcgo.github.io/monty/v0.1.0/schema.json"
+CURRENT_SCHEMA_MAPURL = "https://raw.githubusercontent.com/IFRCGo/monty-stac-extension/refs/heads/main/json-schema/schema.json"
+
+
+def load_scenarios(
+    scenarios: List[str],
+    timeout: int=30
+):
+    transformers = []
+    for scenario_url in scenarios:
+        response = requests.get(scenario_url, timeout=timeout)
+        response_data = response.json()
+
+        idu_data_source = IDUDataSource(
+            source_url=scenario_url,
+            data=json.dumps(response_data[:2])
+        )
+        transformers.append(IDUTransformer(idu_data_source))
+    return transformers
+
+
+event_urls = [
+    "https://helix-copilot-prod-helix-media-external.s3.amazonaws.com/external-media/api-dump/idus/2025-01-28-02-00-05/Ktl2z/idus.json"
+]
```

Comment:
Lets add TODO here if we need to move this content to another location at the end.

Reply:
Done